rcl/src/rcl/wait.c

```diff
@@ -156,7 +163,13 @@ rcl_wait_set_fini(rcl_wait_set_t * wait_set)
 {
   rcl_ret_t result = RCL_RET_OK;
   RCL_CHECK_ARGUMENT_FOR_NULL(wait_set, RCL_RET_INVALID_ARGUMENT);
+
   if (__wait_set_is_valid(wait_set)) {
+    rmw_ret_t ret = rmw_destroy_waitset(wait_set->impl->rmw_waitset);
```

Comment:
It does; I will add it.

Is there an appropriate return code besides `RCL_RET_ERROR` for this case? I was considering `RCL_RET_WAIT_SET_INVALID`